rust/src/smb/smb1.rs

```diff
@@ -732,7 +734,7 @@ fn smb1_response_record_one(state: &mut SMBState, r: &SmbRecord, command: u8, an
                             fid.extend_from_slice(&u32_as_bytes(r.ssn_id));
                             SCLogDebug!("SMB1_COMMAND_NT_CREATE_ANDX fid {:?}", fid);
                             SCLogDebug!("fid {:?} name {:?}", fid, p);
-                            state.guid2name_map.insert(fid, p);
+                            _ = state.guid2name_map.put(fid, p);
```

Comment:
Should we handle somehow the return value ?

Reply:
Not sure what to do with it. IIRC it returns the evicted value.